### PR TITLE
Add com.maxizamorano.photopea-no-ads

### PR DESCRIPTION
**Pull request by MaxiZamorano**
**Name:** Photopea no Ads

**Description:** In addition to removing the ads from Photopea, it also removes the box that the ads occupy so that the work area is not reduced.

### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    build-commands:
+      - install -D -t /app/bin photopea_no_ads.sh
+      - mkdir /app/main
+      - tar -xzf photopea_no_ads_build.tar.gz -C /app/main
+      - install -Dm755 /app/main/resources/data/com.maxizamorano.photopea-no-ads.desktop -t /app/share/applications
+      - install -Dm644 /app/main/resources/data/com.maxizamorano.photopea-no-ads.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 /app/main/resources/icons/photopea.png /app/share/icons/hicolor/512x512/apps/com.maxizamorano.photopea-no-ads.png
+    sources:
+      - type: file
+        url: https://github.com/MaxiZamorano/photopea-no-ads/releases/download/v1.0.0/photopea_no_ads_build.tar.gz
```
Comment (hfiguiere) [inline com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml:36]: how does that work on aarch64?

File: com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: photopea_no_ads.sh
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml:13]: it's either wayland with fallback-x11 or x11 alone. not both.

**Comment by bbhtt** [comment]
The icon being used here is an exact copy of the original icon used by [Photopea](https://www.photopea.com/) https://github.com/MaxiZamorano/photopea-no-ads/blob/main/icons/photopea.png

You cannot do that. You cannot also use someone else's brand name for your application, as such it cannot be named "Photopea no ads" https://github.com/MaxiZamorano/photopea-no-ads/blob/981e06566e6ba3f434655812b521230b1bc3874d/data/com.maxizamorano.photopea-no-ads.metainfo.xml#L5

See https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#no-trademark-violations

File: com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml (hunk 1 above)
Comment (bbhtt) [reply com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml:36]: This should be built from source.

Comment (MaxiZamorano) [reply com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml:35]: > ¿Cómo funciona eso en Aarch64?

I haven't tested it on aarch64, only x86_64.

**Comment by MaxiZamorano** [comment]
> The icon being used here is an exact copy of the original icon used by [Photopea](https://www.photopea.com/) https://github.com/MaxiZamorano/photopea-no-ads/blob/main/icons/photopea.png
> 
> You cannot do that. You cannot also use someone else's brand name for your application, as such it cannot be named "Photopea no ads" https://github.com/MaxiZamorano/photopea-no-ads/blob/981e06566e6ba3f434655812b521230b1bc3874d/data/com.maxizamorano.photopea-no-ads.metainfo.xml#L5
> 
> See https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#no-trademark-violations
> 
> I'm also a bit concerned whether this is acceptable for Flathub or not (but we'll have a discussion about that later)

There is currently a Photopea app on Flathub but it doesn't block ads, it also uses a Photopea icon and the Photopea name, what's the difference? I also see other apps on Flathub with brand names and icons.

**Comment by bbhtt** [comment]
Two wrongs don't make a right. We reserve the right to remove them from the store at any point.
The issue is being tracked https://github.com/flathub/flathub/issues/5228.

File: com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml (hunk 1 above)
Comment (hfiguiere) [reply com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml:35]: but an aarch64 package is still built with these x86_64 binaries. Why not building from source?

**Comment by MaxiZamorano** [comment]
> Two wrongs don't make a right. We reserve the right to remove them from the store at any point. The issue is being tracked #5228.

I understand, so I should modify icon, name and description? ID too?

**Comment by bbhtt** [comment]
Name and icon, yes. The description should describe your app, not Photopea.

**Pull request closed without merging**